pytensor/tensor/slinalg.py

```diff
@@ -129,8 +132,10 @@ def conjugate_solve_triangular(outer, inner):
             return [grad]
 
 
-def cholesky(x, lower=True, on_error="raise"):
-    return Blockwise(Cholesky(lower=lower, on_error=on_error))(x)
+def cholesky(x, lower=True, on_error="raise", check_finite=True):
```

Comment:
What about default check_finite=False?

Deviates from Scipy, but avoids iterating over the array before every call. Since this wasn't a problem until now and JAX probably just drops the flag anyway, it doesn't seem too dangerous?

It also doesn't sound like it will lead to any silent bad outputs.

Reply:
Works for me